xm-commons-topic/src/main/java/com/icthh/xm/commons/topic/util/MessageHeaderUtils.java

```diff
@@ -0,0 +1,76 @@
+package com.icthh.xm.commons.topic.util;
+
+import com.icthh.xm.commons.logging.util.MdcUtils;
+import lombok.experimental.UtilityClass;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.common.header.Header;
+import org.apache.kafka.common.header.Headers;
+
+import java.math.BigInteger;
+import java.util.Date;
+
+@UtilityClass
+public class MessageHeaderUtils {
+
+    private static final String RID_HEADER = "rid";
```

Comment:
maybe it is worth to add `xm_` prefix to headers `rid`,  `retry` and `startProcessingTime` to make them more XM specific